.github/ISSUE_TEMPLATE/late_release_exception.yml

```diff
@@ -0,0 +1,75 @@
+# This issue template is for use in opening issues that document late release exception requests.
+# 
+# Bracketed text is a placeholder; replace the text with the requested information and remove the brackets before submitting the issue. Also, remove this comment before submitting the issue.
+
+name: "Identity Model Late Release Exception Request"
+description: Request an exception to release a feature after the release candidate deadline.
```

Comment:
```suggestion
description: Request an exception to release a feature or add any changes after the release candidate deadline, excludes fixes for preview regressions.
```

Reply:
It won't let me batch your suggestions for some reason so I just pushed one commit taking all of your suggestions. Thank you!

![image](https://github.com/user-attachments/assets/f8fd92c1-b97c-4ecb-9c3f-3ed1164c1d65)